.goreleaser.docker.yaml

```diff
@@ -0,0 +1,250 @@
+      - '--label=org.opencontainers.image.name=jvs-justification'
+      - '--label=org.opencontainers.image.revision={{ .FullCommit }}'
+      - '--label=org.opencontainers.image.source={{ .GitURL }}'
+      - '--label=org.opencontainers.image.title=jvs-justification'
+      - '--label=org.opencontainers.image.version={{ .Version }}'
+
+  -
+    ids:
+    - public-key
+    use: 'buildx'
+    goos: 'linux'
+    goarch: 'amd64'
+    image_templates:
+      - '{{ .Env.REGISTRY }}/jvs-public-key:{{ .Version }}-amd64'
+    build_flag_templates:
+      - '--build-arg=APP=public-key'
+      - '--platform=linux/amd64'
+      - '--pull'
+      - '--label=org.opencontainers.image.created={{ .CommitTimestamp }}'
+      - '--label=org.opencontainers.image.description=JVS public key is a HTTP service that exposes the public keys as JWKs for validating justifcation tokens.'
+      - '--label=org.opencontainers.image.licenses=Apache-2.0'
+      - '--label=org.opencontainers.image.name=jvs-public-key'
+      - '--label=org.opencontainers.image.revision={{ .FullCommit }}'
+      - '--label=org.opencontainers.image.source={{ .GitURL }}'
+      - '--label=org.opencontainers.image.title=jvs-public-key'
+      - '--label=org.opencontainers.image.version={{ .Version }}'
+  -
+    ids:
+    - public-key
+    use: 'buildx'
+    goos: 'linux'
+    goarch: 'arm64'
+    image_templates:
+      - '{{ .Env.REGISTRY }}/jvs-public-key:{{ .Version }}-arm64'
+    build_flag_templates:
+      - '--build-arg=APP=public-key'
+      - '--platform=linux/arm64'
+      - '--pull'
+      - '--label=org.opencontainers.image.created={{ .CommitTimestamp }}'
+      - '--label=org.opencontainers.image.description=JVS public key is a HTTP service that exposes the public keys as JWKs for validating justifcation tokens.'
+      - '--label=org.opencontainers.image.licenses=Apache-2.0'
+      - '--label=org.opencontainers.image.name=jvs-public-key'
+      - '--label=org.opencontainers.image.revision={{ .FullCommit }}'
+      - '--label=org.opencontainers.image.source={{ .GitURL }}'
+      - '--label=org.opencontainers.image.title=jvs-public-key'
+      - '--label=org.opencontainers.image.version={{ .Version }}'
+
+
+docker_manifests:
+  -
+    name_template: '{{ .Env.REGISTRY }}/jvs-cert-rotation:{{ .Version }}'
+    image_templates:
+      - '{{ .Env.REGISTRY }}/jvs-cert-rotation:{{ .Version }}-amd64'
+      - '{{ .Env.REGISTRY }}/jvs-cert-rotation:{{ .Version }}-arm64'
+  -
+    name_template: '{{ .Env.REGISTRY }}/jvs-justification:{{ .Version }}'
+    image_templates:
+      - '{{ .Env.REGISTRY }}/jvs-justification:{{ .Version }}-amd64'
+      - '{{ .Env.REGISTRY }}/jvs-justification:{{ .Version }}-arm64'
+  -
+    name_template: '{{ .Env.REGISTRY }}/jvs-public-key:{{ .Version }}'
+    image_templates:
+      - '{{ .Env.REGISTRY }}/jvs-public-key:{{ .Version }}-amd64'
+      - '{{ .Env.REGISTRY }}/jvs-public-key:{{ .Version }}-arm64'
+
+
+# TODO(#129): cosign the images.
```

Comment:
Now as we separate the two type release, it seems to make more sense to cosign images?

Reply:
Let's move that into the follow-up. goreleaser can also do sboms, checksumming, gpging, etc. I think we want to dive in a bit more and maybe check with customers on what'd they'd expect.

Reply:
Updated the issue link